library/src/main/java/com/android/volley/toolbox/DiskBasedCache.java

```diff
@@ -610,7 +609,7 @@ static void writeStringStringMap(Map<String, String> map, OutputStream os) throw
         static Map<String, String> readStringStringMap(InputStream is) throws IOException {
             int size = readInt(is);
             Map<String, String> result = (size == 0)
-                    ? Collections.<String, String>emptyMap()
+                    ? new HashMap<String, String>()
```

Comment:
Why do we even bother to construct a hash map with a set capacity? HashMap will auto double when it needs more space so...?